freqtrade/rpc/telegram.py

```diff
@@ -466,13 +466,16 @@ def _help(self, bot: Bot, update: Update) -> None:
         :param update: message update
         :return: None
         """
+        forcebuy_text = "*/forcebuy <pair> [<rate>]:* `Instantly buys the given pair. " \
+                        "Optionally takes a rate at which to buy.` \n"
         message = "*/start:* `Starts the trader`\n" \
                   "*/stop:* `Stops the trader`\n" \
                   "*/status [table]:* `Lists all open trades`\n" \
                   "         *table :* `will display trades in a table`\n" \
                   "*/profit:* `Lists cumulative profit from all finished trades`\n" \
                   "*/forcesell <trade_id>|all:* `Instantly sells the given trade or all trades, " \
                   "regardless of profit`\n" \
+                  f"{forcebuy_text if self._config.get('forcebuy_enable', False) else '' }" \
```

Comment:
any suggestion how we can improve the messaging without these if and else ? I want to add bunch of conditional things too but it would create a mess :/

Reply:
what exactly? remember that this is the /help command ... i don't know any other optional options at the moment ...